src/lib/make_descr.c

```diff
@@ -236,6 +233,10 @@ char *libreport_make_description(problem_data_t *problem_data, char **names_to_s
 
     if (desc_flags & MAKEDESC_SHOW_MULTILINE)
     {
+        const char *type = problem_data_get_content_or_NULL(problem_data,
+                                                            FILENAME_TYPE);
+        bool is_kernel_oops = type != NULL && strcmp(type, "Kerneloops") == 0;
```

Comment:
Just as an FYI: `g_strcmp0()` handles null strings, so you could save on typing, but this is fine, too.

Reply:
Actually, I'll change that. Thanks for the tip.